fc/parsing/actions.py

```diff
@@ -982,10 +982,10 @@ def _annotate_state_variables(self):
         state_annotation = create_rdf_node((OXMETA_NS, 'state_variable'))
         self.vector_orderings[state_annotation] = {}
         for i, state_var in enumerate(self.model.get_state_symbols()):
-            if not state_var.cmeta_id:
-                state_var.cmeta_id = self.model.get_unique_cmeta_id(state_var.name.replace('$', '__'))
+            if not state_var.rdf_identity:
```

Comment:
We can remove the `if` here, and just do `self.model.add_cmeta_id(state_var)` since it's a no-op if one exists.